pipelines/controller/src/main/resources/templates/index.html

```diff
@@ -232,6 +250,9 @@ <h2 class="accordion-header" id="flush-headingOne">
             </div>
           </div-->
         </div>
+        <div class="alert alert-success" style="float:right;display:none;" id="success">
+          Request for resource tables creation sent successfully.
```

Comment:
Please also handle the error, i.e., if the API call fails for whatever reason, please communicate it in the UI (preferrably with a relevant error message but at least a generic error message).